convert2rhel/actions/pre_ponr_changes/custom_repos_are_valid.py

```diff
@@ -37,7 +38,7 @@ def run(self):
         super(CustomReposAreValid, self).run()
         logger.task("Prepare: Check if --enablerepo repositories are accessible")
 
-        if not tool_opts.no_rhsm:
+        if not tool_opts.enablerepo:
             logger.info("Did not perform the check of repositories due to the use of RHSM for the conversion.")
```

Comment:
The log message does not match the condition anymore. I'll create a PR for that.